pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -188,6 +188,7 @@ func (c *Reconciler) reconcileService(el *v1alpha1.EventListener) error {
 			return err
 		} else {
 			el.Status.Configuration.Hostname = listenerHostname(service.Name, el.Namespace)
```

Comment:
We could remove this line I suppose and migrate the tests to this Addressable

Reply:
done!